io-pay-portal-fe/src/components/FormButtons/FormButtons.tsx

```diff
@@ -52,17 +48,16 @@ export function FormButtons(props: {
               width: "100%",
               height: "100%",
               minHeight: 45,
-              padding: "10px 24px",
             }}
           >
             {t(props.cancelTitle)}
           </Button>
         </Grid>
         <Grid
-          xs={6}
-          md={6}
-          lg={6}
-          xl={6}
+          xs={8}
+          md={8}
+          lg={8}
+          xl={8}
           style={useSmallDevice() ? { paddingTop: 0 } : {}}
```

Comment:
Could you write this in the pattern expected avoing style? Like
`pt={xs: 1, sm: 0}`

Reply:
on xs screen pt={xs: 1, sm: 0} doesn't overwrite the mui grid item default padding of 16px, while it does with style. 
will try a better solution.